### PR TITLE
    HMAN-217: Round up duration in multi-day hearing to nearest multi…

### PR DESCRIPTION
**Pull request by Thor-tech-of-metal**
HMAN-217: Round up duration in multi-day hearing to nearest multiple of 5
              [HMAN-217] (https://tools.hmcts.net/jira/browse/HMAN-217).

### CODE REVIEW

File: build.gradle [hunk 1]
```diff
@@ -441,8 +441,9 @@ dependencies {
   integrationTestImplementation sourceSets.main.runtimeClasspath
   integrationTestImplementation sourceSets.test.runtimeClasspath
   integrationTestImplementation group: 'net.minidev', name: 'json-smart', version: '2.4.7'
-
   integrationTestImplementation group: 'org.testcontainers', name: 'postgresql', version: versions.testcontainers
+  integrationTestImplementation group: 'org.projectlombok', name: 'lombok', version: versions.lombok
```
Comment (sullivangareth) [inline build.gradle:445]: Is importing lombok dependency just to be able to use `val` keyword worth it ?

It also makes code read bit inconsistent ?

File: src/test/java/uk/gov/hmcts/reform/hmc/helper/hmi/ListingMapperTest.java [hunk 2]
```diff
@@ -272,4 +268,57 @@ private void generateOtherConsiderations(List<String> facilityTypes, ListingJoh
...
+        assertEquals(DURATION_OF_DAY, listing.getListingDuration());
+        assertEquals(0, listing.getListingMultiDay().getWeeks());
+        assertEquals(1, listing.getListingMultiDay().getDays());
+        assertEquals(3, listing.getListingMultiDay().getHours());
+    }
+
+    private Listing getListing(int duration) {
+        val hearingMapper =
+            new HearingMapper(null, null, null);
+        HearingDetails hearingDetails = buildHearingDetails(hearingMapper.roundUpDuration(duration));
```
[the remark this answers is not in the document]
Comment (sullivangareth) [reply src/test/java/uk/gov/hmcts/reform/hmc/helper/hmi/ListingMapperTest.java:320]: Or put it in a utility class or something ?

File: src/test/java/uk/gov/hmcts/reform/hmc/helper/hmi/ListingMapperTest.java [hunk 3]
```diff
@@ -272,4 +268,57 @@ private void generateOtherConsiderations(List<String> facilityTypes, ListingJoh
         when(listingOtherConsiderationsMapper.getListingOtherConsiderations(any(), any()))
             .thenReturn(otherConsiderations);
     }
+
+    @Test
+    void shouldReturnListingForMultiDayHearingDurationWithManyValues() {
```
[the remark this answers is not in the document]
Comment (Thor-tech-of-metal) [reply src/test/java/uk/gov/hmcts/reform/hmc/helper/hmi/ListingMapperTest.java:273]: Yes, I need to re-factor this test . good point .    @ParameterizedTest(value=“) mm that can be used for external parameters. I would keep the test data in the test body. DONE

File: src/test/java/uk/gov/hmcts/reform/hmc/helper/hmi/ListingMapperTest.java (hunk 2 above)
[the remark this answers is not in the document]
Comment (Thor-tech-of-metal) [reply src/test/java/uk/gov/hmcts/reform/hmc/helper/hmi/ListingMapperTest.java]: good point I make it static.

File: src/test/java/uk/gov/hmcts/reform/hmc/helper/hmi/ListingMapperTest.java (hunk 3 above)
[the remark this answers is not in the document]
Comment (Thor-tech-of-metal) [reply src/test/java/uk/gov/hmcts/reform/hmc/helper/hmi/ListingMapperTest.java:273]: Yes, I need to re-factor this test . good point . @ParameterizedTest(value=“) mm that can be used for external parameters. I would keep the test data in the test body. DONE

File: build.gradle (hunk 1 above)
Comment (Thor-tech-of-metal) [reply build.gradle:445]: Lombok is nice, val is good . :) hehe

File: src/test/java/uk/gov/hmcts/reform/hmc/helper/HearingMapperTest.java [hunk 4]
```diff
@@ -0,0 +1,35 @@
...
+
+    @Mock
+    private PartyDetailMapper partyDetailMapper;
+
+    @Mock
+    private HearingDetailsMapper hearingDetailsMapper;
+
+
+    @Test
+    void modelToEntityTest() {
```
[the remark this answers is not in the document]
Comment (Thor-tech-of-metal) [reply src/test/java/uk/gov/hmcts/reform/hmc/helper/HearingMapperTest.java:22]: Yes, I need to re-factor this test . good point . @ParameterizedTest(value=“) mm that can be used for external parameters. I would keep the test data in the test body.

File: build.gradle (hunk 1 above)
Comment (sullivangareth) [reply build.gradle:445]: I don't like it, but again guess it's down to personal preference

**Review by sullivangareth — APPROVED** [review]

**Pull request merged**